components/navigation/NavBar.js

```diff
@@ -50,6 +50,12 @@ export default function NavBar({
     setOpen(null);
   }, [asPath])
 
+  useEffect(() => {
+    mobileMenuOpen ? 
+    document.body.style.overflow = 'hidden' :
+    document.body.style.overflow = 'auto';
+  }, [mobileMenuOpen])
```

Comment:
Error is only related to z-index, don't add other properties related to the body of the website.

Reply:
Body shouldn't be scrollable upon opening of nav-bar right?

Reply:
Yeah, but isn't a good practice to directly add properties using DOM for body tag, React always renders the data using `root` element, not the body tag. Also, I don't think if navbar gets scrolled in the background, it won't be affecting the UI experience at greater rate, so you shouldn't add the properties to body in this way.

Reply:
Done.